modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/IgniteSqlFunctions.java

```diff
@@ -42,6 +42,15 @@
  * Ignite SQL functions.
  */
 public class IgniteSqlFunctions {
+    /** */
+    public static final String NUMERIC_OVERFLOW_ERROR = "Numeric field overflow.";
+
+    /** */
+    private static final int DFLT_NUM_PRECISION = IgniteTypeSystem.INSTANCE.getDefaultPrecision(SqlTypeName.DECIMAL);
+
+    /** */
+    private static final RoundingMode NUMERIC_ROUNDING_MODE = RoundingMode.HALF_UP;
```

Comment:
We use CEILING rounding mode, but for some reason tests are passed with this rounding mode when I add cases like convertation from 100.16 to 100.1.

Reply:
Didn't get. We use HALF_UP